app/scripts/controllers/network/network-controller.test.js

```diff
@@ -50,224 +69,4106 @@ const PRE_1559_BLOCK = {
     '0x98bbdfbe1074bc3aa72a77a281f16d6ba7e723d68f15937d80954fb34d323369',
   uncles: [],
 };
-// Example block taken from Ethereum Mainnet post-EIP-1559
-const BLOCK = {
+
+/**
+ * A dummy block that matches the pre-EIP-1559 format (i.e. it has the
+ * `baseFeePerGas` property).
+ */
+const POST_1559_BLOCK = {
   ...PRE_1559_BLOCK,
   baseFeePerGas: '0x63c498a46',
 };
 
-const defaultControllerOptions = {
-  infuraProjectId: 'foo',
+/**
+ * An alias for `POST_1559_BLOCK`, for tests that don't care about which kind of
+ * block they're looking for.
+ */
+const BLOCK = POST_1559_BLOCK;
+
+/**
+ * A dummy value for the `projectId` option that `createInfuraClient` needs.
+ * (Infura should not be hit during tests, but just in case, this should not
+ * refer to a real project ID.)
+ */
+const DEFAULT_INFURA_PROJECT_ID = 'infura-project-id';
```

Comment:
NIT: use a very obviously incorrect value 
```suggestion
const DEFAULT_INFURA_PROJECT_ID = 'fake-infura-project-id';
```